deploy/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.access.redhat.com/ubi7-minimal
```

Comment:
Any reason why we don't use UBI8 ?

Reply:
I implicitly associated os311 with rhel7 and ubi7. We probably should investigate this.

Reply:
So since we're running on OS3/RHEL7, the best supported container host would be UBI7.
My question got answered